stdlib/public/runtime/SymbolInfo.h

```diff
@@ -43,14 +43,74 @@ namespace swift {
 struct SymbolInfo {
 private:
 #if defined(_WIN32) && !defined(__CYGWIN__)
-  SYMBOL_INFO_PACKAGE _package;
```

Comment:
What's your thinking in not storing the whole `SYMBOL_INFO_PACKAGE` here? Less transient memory?

Reply:
Partly that. Also, when `SymFromAddr()` fails, I wanted to still give the module name and base address, as well as indicating the address we were looking at (but obviously no symbol).

Reply:
It's swings and roundabouts though, because of the `::malloc()`s.